src/content/projects/yordle.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: "/projects/yordle"
+date: "2022-06-27"
+title: "Yordle"
+preview: "../../images/Projects/Yordle/preview.png"
+client: "Yordevs"
+description: "After the rise in popularity of 'Wordle', we created a clone of the game where every answer was in some way related to York"
+lead: "Adam Barr"
+developers: ["Ben Silverman", "Harry Wickham"]
+link: "https://yordle.yordevs.com"
+---
+
+---
+
+# Yordle
+
+Wordle took the internet by storm over the first few months of 2022, whilst the idea behind the game was easy to understand, completing the a wordle provided a sense of accomplishment and pride you couldn't help but show of to your friends and family. It was a perfect mix for a viral game.
```

Comment:
Wordle took the internet by storm over the first few months of 2022, whilst the idea behind the game was easy to understand, completing _**a**_ wordle provided a sense of accomplishment and pride you couldn't help but show _**off**_ to your friends and family. It was a perfect mix for a viral game.